staging/knative/charts/serving/values.yaml

```diff
@@ -7,3 +7,6 @@ customMetricsApiservice:
 
 namespaceKnativeServing:
   additionalLabels: {}
+
+deployment:
```

Comment:
There multiple deployment kind in knativer serving. To be explicit, let's rename it as `configDeployment` or `deploymentConfigMap`?

Reply:
renamed to `configDeployment`